docs/university.md

```diff
@@ -2088,6 +2088,59 @@ jsjxy.hbut.edu.cn 证书链不全，自建 RSSHub 可设置环境变量 NODE_TLS
 
 </Route>
 
+## 南京艺术学院
```

Comment:
The same H2 heading already appeared in https://github.com/DIYgod/RSSHub/blob/8026e17f26bc431e5b2101872959960d420471b6/docs/university.md?plain=1#L2052
Do not add it again as it will [break the docs](https://github.com/DIYgod/RSSHub/runs/7986042058?check_suite_focus=true#step:5:22)